hw03-annotations/src/main/java/ru/eon/Main.java

```diff
@@ -0,0 +1,69 @@
+package ru.eon;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Main {
+    private static final Logger log = LoggerFactory.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        runTests(TestClass.class);
```

Comment:
Результат здесь лучше возвращать.
```suggestion
        StatisticOfTests result = runTests(TestClass.class);
        log.info("Total tests {}", result.getTotalTests());
        log.info("Total tests succeed {}", result.getTotalSuccess());
        log.info("Total tests failure {}", result.getTotalFailure());
```

И `StatisticOfTests` сделать неизменяемым (без `increaseTotalTests`, `increaseTotalSuccess`).